syntax = "proto3";

package echo;


option cc_generic_services = true;

message EchoRequest {
    optional string msg = 1;
}

message EchoResponse {
    optional string msg = 2;
}


message GetDataReq {
    optional string msg = 1;
}
message GetDataRsp {
    optional string msg = 1;
}

service EchoService {
    rpc Echo(EchoRequest) returns (EchoResponse);
    rpc Echo2(EchoRequest) returns (EchoResponse);
    rpc GetData(GetDataReq) returns (GetDataRsp);
}
